cpp/include/raft/neighbors/detail/cagra/hashmap.hpp

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Is it possible to use the hashmap from cuCollections for this https://github.com/NVIDIA/cuCollections - since we already have this as a dependency?